editor/editor_themes.cpp

```diff
@@ -162,37 +161,29 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 			base_color = Color::html("#383c4a");
 			contrast = 0.25;
 		} break;
+		case 4: { // Light
+			highlight_color = Color::html("#2070ff");
+			base_color = Color::html("#efefef");
+			contrast = 0.04;
+		} break;
 	}
 
+	//Colors
+	bool dark_theme = ((base_color.r + base_color.g + base_color.b) / 3.0) < 0.5;
```

Comment:
I think this should be made an option, with three values: Auto (same as here and default value), Black, White (or Dark, Light, as you want).

People have different eyes and different tastes, and not everyone might be happy with the hardcoded limit of the auto behaviour.

Reply:
so doing:
```
bool dark_theme = icon_color_setting == DARK || (icon_color_setting == AUTO && ((base_color.r + base_color.g + base_color.b) / 3.0) < 0.5 );
```

Reply:
im not sure about this one, i think it has to be automatic because of the contrast also it would add another step to normal people who cant see white on light grey.

Reply:
That's why I propose to have an Auto mode which is the default value, and behaves as implemented already. But I do think that we'd have users trying a greyish theme and not happy that they can't decide by themselves if they want the dark icon set + font color or the light one (in cases where both might be contrasted enough, or in cases where the auto logic might not pick the best setting).